libs/log/testing.go

```diff
@@ -1,43 +1,18 @@
 package log
 
 import (
-	"sync"
 	"testing"
 
 	"github.com/rs/zerolog"
 )
 
-var (
-	// reuse the same logger across all tests
-	testingLoggerMtx = sync.Mutex{}
-	testingLogger    Logger
-)
-
-// TestingLogger returns a Logger which writes to STDOUT if test(s) are being
-// run with the verbose (-v) flag, NopLogger otherwise.
-//
-// NOTE:
-// - A call to NewTestingLogger() must be made inside a test (not in the init func)
-// because verbose flag only set at the time of testing.
-// - Repeated calls to this function within a single process will
-// produce a single test log instance, and while the logger is safe
-// for parallel use it it doesn't produce meaningful feedback for
-// parallel tests.
+// TestingLogger was a legacy constructor that wrote logging output to
+// standardoutput when in verbose mode, and no-op'ed test logs
+// otherwise. Now it always no-ops, but if you need to see logs from
+// tests, you can replace this call with `NewTestingLogger`
+// constructor.
```

Comment:
Maybe also:
```suggestion
// constructor.
//
// Deprecated: Use New NopLogger or NewTestingLogger for new tests.
```

which will get staticcheck to yell at us if we add new uses (but I believe should not complain about existing use).